src/shared/services/embed-code-service.ts

```diff
@@ -56,4 +39,31 @@ export class EmbedCodeService {
 			});
 		}
 	}
+
+	public static async getEmbedCodes(params?: EmbedCodeFilters): Promise<{
+		embedCodes: EmbedCode[];
+		count: number;
+	}> {
+		let url: string | undefined = undefined;
+
+		try {
+			url = stringifyUrl({
+				url: `${getEnv('PROXY_URL')}/embed-codes`,
+				query: {
+					sortColumn: params?.sortColumn,
+					sortOrder: params?.sortOrder,
+					tableColumnDataType: params?.sortType,
```

Comment:
if we build the order object on the proxy then we don't need to pass this type to the proxy
```suggestion
```